### PR TITLE
Add type annotations to ``bioblend.galaxy.genomes``

### PR DESCRIPTION
**Pull request by thepineapplepirate**
added typing info, and made the other necessary changes to /bioblend/galaxy/genomes/__init__.py

### CODE REVIEW

File: bioblend/galaxy/genomes/__init__.py [hunk 1]
```diff
@@ -20,7 +32,7 @@ def get_genomes(self):
         genomes = self._get()
         return genomes
 
-    def show_genome(self, id, num=None, chrom=None, low=None, high=None):
+    def show_genome(self, id: str, num: str = None, chrom: str = None, low: str = None, high: str = None) -> Response:
```
Comment (nsoranzo) [inline bioblend/galaxy/genomes/__init__.py:35]:
```suggestion
    def show_genome(self, id: str, num: str = None, chrom: str = None, low: str = None, high: str = None) -> dict:
```

File: bioblend/galaxy/genomes/__init__.py [hunk 2]
```diff
@@ -51,17 +63,17 @@ def show_genome(self, id, num=None, chrom=None, low=None, high=None):
             params["low"] = low
         if high:
             params["high"] = high
-        return self._get(id, params)
+        return self._get(id=id, params=params)
```
Comment (nsoranzo) [inline bioblend/galaxy/genomes/__init__.py:66]: Thanks for the fix, that was broken!

Comment (thepineapplepirate) [reply bioblend/galaxy/genomes/__init__.py:66]: yeap, thanks to Fabio! it kept asking me for a boolean here, and it didn't make any sense to me.

**Review by nsoranzo — APPROVED** [review]

File: bioblend/galaxy/genomes/__init__.py (hunk 2 above)
Comment (nsoranzo) [reply bioblend/galaxy/genomes/__init__.py:66]: Group work :heart:

File: bioblend/galaxy/quotas/__init__.py [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
 
 class QuotaClient(Client):
     module = "quotas"
 
-    def __init__(self, galaxy_instance):
+    def __init__(self, galaxy_instance: "GalaxyInstance") -> None:
         super().__init__(galaxy_instance)
 
-    def get_quotas(self, deleted=False):
+    def get_quotas(self, deleted=False) -> List[Any]:
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:24]:
```suggestion
    def get_quotas(self, deleted=False) -> List[Dict[str, Any]]:
```

File: bioblend/galaxy/quotas/__init__.py [hunk 4]
```diff
@@ -32,7 +43,7 @@ def get_quotas(self, deleted=False):
         """
         return self._get(deleted=deleted)
 
-    def show_quota(self, quota_id, deleted=False):
+    def show_quota(self, quota_id: str, deleted=False) -> Dict[str, Any]:
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:46]:
```suggestion
    def show_quota(self, quota_id: str, deleted: bool = False) -> Dict[str, Any]:
```

File: bioblend/galaxy/quotas/__init__.py [hunk 5]
```diff
@@ -59,7 +70,16 @@ def show_quota(self, quota_id, deleted=False):
         """
         return self._get(id=quota_id, deleted=deleted)
 
-    def create_quota(self, name, description, amount, operation, default="no", in_users=None, in_groups=None):
+    def create_quota(
+        self,
+        name: str,
+        description: str,
+        amount: str,
+        operation: str,
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:78]:
```suggestion
        operation: QuotaOperations,
```

File: bioblend/galaxy/quotas/__init__.py [hunk 6]
```diff
@@ -59,7 +70,16 @@ def show_quota(self, quota_id, deleted=False):
...
+        default: str = "no",
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:79]:
```suggestion
        default: Optional[Literal["no", "registered", "unregistered"]] = "no",
```

File: bioblend/galaxy/quotas/__init__.py [hunk 7]
```diff
@@ -1,16 +1,27 @@
 """
 Contains possible interactions with the Galaxy Quota
 """
+
+from typing import (
+    Any,
+    Dict,
+    List,
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:8]:
```suggestion
    List,
    Literal,
```

File: bioblend/galaxy/quotas/__init__.py [hunk 8]
```diff
@@ -1,16 +1,27 @@
...
+    TYPE_CHECKING,
+)
+
 from bioblend.galaxy.client import Client
 
+if TYPE_CHECKING:
+    from bioblend.galaxy import GalaxyInstance
+
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:16]:
```suggestion
if TYPE_CHECKING:
    from bioblend.galaxy import GalaxyInstance

QuotaOperations = Literal["+", "-", "="]

```

File: bioblend/galaxy/quotas/__init__.py [hunk 9]
```diff
@@ -113,15 +133,16 @@ def create_quota(self, name, description, amount, operation, default="no", in_us
...
-        operation=None,
-        default="no",
-        in_users=None,
-        in_groups=None,
-    ):
+        quota_id: str,
+        name: str = None,
+        description: str = None,
+        amount: str = None,
+        operation: str = None,
```
Comment (nsoranzo) [inline bioblend/galaxy/quotas/__init__.py:140]:
```suggestion
        operation: QuotaOperations = None,
```

File: bioblend/galaxy/genomes/__init__.py [hunk 10]
```diff
@@ -1,16 +1,28 @@
...
 """
+from typing import (
+    Any,
+    Dict,
+    List,
+    TYPE_CHECKING,
+)
+
+from requests.models import Response
+
```
Comment (nsoranzo) [inline bioblend/galaxy/genomes/__init__.py:12]:
```suggestion
```

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @thepineapplepirate !

**Pull request merged**